images/kong/plugins/0.12.3/jwt/handler.lua

```diff
@@ -114,7 +114,9 @@ local function do_authentication(conf)
   -- Decode token to find out who the consumer is
   local jwt, err = jwt_decoder:new(token)
   if err then
-    return false, {status = 401, message = "Bad token; " .. tostring(err)}
+    -- Don't sent Bad Token for null / empty Bearer tokens
+    -- return false, {status = 401, message = "Bad token; " .. tostring(err)}
```

Comment:
remove this commented line @keshavprasadms

Reply:
@rjshrjndrn Removed that. Also changed kong tag.